### PR TITLE
Bulk stack transfer blog post

### PR DESCRIPTION
**Pull request by meagancojocar**

### CODE REVIEW

File: themes/default/content/blog/stack-transfers/index.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+# for additional details, and please remove these comments before submitting for review.
+---
+
+Exactly 3 years ago we added support in the [Pulumi Service](/product/pulumi-service) to transfer stacks from an Individual account to a Pulumi organization and between Pulumi organizations. We heard from customers that they love this feature but found it both hard to discover and tedious when moving a large workload from one organization to another and from Individual accounts to organizations. We are excited to announce bulk stack transfer to address this feedback and a new organization set up wizard to improve discovery of the feature.
+
+<!--more-->
+
+Let’s review how bulk stack transfers can be used and when you would use them. Stack transfers relocate a stack between organizations and Individual accounts. Let’s highlight some use cases:
+
+- **Use case #1: Individual account to a collaborative organization:** We know that users are building projects in Individual accounts and want to easily move their stacks to an organization so they can collaboratively build infrastructure with others
```
Comment (gthuang) [inline themes/default/content/blog/stack-transfers/index.md:42]:
```suggestion
- **Use case #1: Individual account to a collaborative organization:** We know that users are building projects in Individual accounts and want to easily move their stacks to an organization so they can collaboratively build infrastructure with others.
```

File: themes/default/content/blog/stack-transfers/index.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+- **Use case #2: Organization to Individual account:** There may be instances where users want to take a stack from an organization into their Individual org. This could be due to mistakenly creating a stack in the wrong location.
+- **Use case #3: Organization to organization:** As customers scale there may be use cases where they want to add additional organizations and being able to transfer stacks in bulk will help with this.
+
+### Transferring one stack
+
+To transfer a specific stack, you can navigate to that stack and then go to Settings > Transfer Stack. To transfer a stack, you need to be either the Stack Admin or an Organization Admin. For organizations on the Pulumi Enterprise or Business Critical Editions, an audit log will be produced each time a stack is transferred and it contains which organization the stack was transferred to.
```
Comment (gthuang) [inline themes/default/content/blog/stack-transfers/index.md:48]:
```suggestion
To transfer a specific stack, you can navigate to that stack and then go to Settings > Transfer Stack. To transfer a stack, you need to be either the Stack Admin or an Organization Admin. For organizations on the Pulumi Enterprise or Business Critical Editions, an audit log will be produced each time a stack is transferred and it records which organization the stack was transferred to.
```

File: themes/default/content/blog/stack-transfers/index.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+![Transfer stack](transfer_stack.png)
+
+### Transferring multiple stacks
+
+To transfer multiple stacks, you can navigate to the Stacks page and then click the `...` icon beside the ‘Create a Project’ button in a collaborative organization and the `Transfer stacks` button in an Individual account. In order to transfer multiple stacks you need to be an Organization Admin. For organizations on the Pulumi Enterprise or Business Critical Editions, an audit log will be produced each time a stack is transferred and it contains which organization the stack was transferred to.
```
Comment (gthuang) [inline themes/default/content/blog/stack-transfers/index.md:54]:
```suggestion
To transfer multiple stacks, you can navigate to the Stacks page and then click the `...` icon beside the ‘Create a Project’ button in a collaborative organization and the `Transfer stacks` button in an Individual account. In order to transfer multiple stacks you need to be an Organization Admin. For organizations on the Pulumi Enterprise or Business Critical Editions, an audit log will be produced each time a stack is transferred and it records which organization the stack was transferred to.
```

File: themes/default/content/blog/stack-transfers/index.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+![Bulk stack transfer](bulk_transfer.png)
+
+### Bringing stacks into new organizations
+
+We recently added a new organization setup wizard that will walk you through inviting members, importing stacks and setting your preferred cloud and language when you create a new organization. This compacts common usage patterns into one place to make it easier for you.
+
+See it in action!
+
+![New org flow gif](new-org-flow.png)
```
Comment (gthuang) [inline themes/default/content/blog/stack-transfers/index.md:64]: is this supposed to be a GIF? it's not moving for me

**Review by gthuang — APPROVED** [review]

File: themes/default/content/blog/stack-transfers/index.md (hunk 4 above)
Comment (meagancojocar) [reply themes/default/content/blog/stack-transfers/index.md:64]: It was! But we have tech limitations on gifs at the moment so I wasn't able to use one. Will edit the image name
```suggestion
![New org flow](new-org-flow.png)
```

**Pull request merged**